### PR TITLE
Add Requirements.txt

### PR DESCRIPTION
**Pull request by jcjgraf**
Implements #15

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+pillow
+numpy
+tensorflow
+tensorflow-gpu
+python-utils
+scipy
+scikit-learn
+scikit-image
+pyyaml
```
Comment (matpalm) [inline requirements.txt:9]: where does the `pyyaml` dep come in from?

Comment (matpalm) [reply requirements.txt:9]: doesn't `tensorflow-gpu` imply `tensorflow` ?

Comment (jcjgraf) [reply requirements.txt:9]: > where does the `pyyaml` dep come in from?

File `util.py`

> doesn't `tensorflow-gpu` imply `tensorflow` ?

Well yes, but only if you use a GPU. For people like me, who do not have a powerful GPU (yet) and hence use the CPU, `tensorflow` is the right one...

Comment (matpalm) [reply requirements.txt:9]: oh yes, util.py, sorry was just grepping in the wrong directory :/

**Comment by matpalm** [comment]
thanks for the contribution!

**Comment by denismedinas** [comment]
Hi Mat,

I'm trying to use the github example of bnn to count bees but I have an error at the training step. 

Traceback (most recent call last):
  File "./train.py", line 132, in <module>
    steps=num_test_steps)
  File "/home/denis/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 960, in evaluate
    callbacks=callbacks)
  File "/home/denis/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training_arrays.py", line 380, in model_iteration
    aggregator.finalize()
  File "/home/denis/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training_utils.py", line 111, in finalize
    raise ValueError('Empty training data.')
ValueError: Empty training data.

thanks,

Denis

**Pull request merged**